.github/ISSUE_TEMPLATE/hosting.md

```diff
@@ -0,0 +1,52 @@
+---
+name: hosting
+about: Wanting to host the meetup? Answer the following questions
+---
+
+Thank you for wanting to host the Ruby Usergroup Hamburg!
+Here we have collected some information regarding hosting the Usergroup.
+Additionally, it would be great if you could answer the questions at the end of the issue. This will help us to have a good overview
+
+# FAQ
```

Comment:
Should the FAQ be a separate page?
Alternatively, we could update the location page on the website and maybe position it more visible

Reply:
whether or not this should be somewhere on the onruby page should be discussed in the onruby project

Reply:
I removed the FAQ part and link to the appropriate places.